src/mainwindow.cpp

```diff
@@ -2932,7 +2932,7 @@ void MainWindow::dropShadow(QPainter& painter, ShadowType type, MainWindow::Shad
  * \param rect
  * \param gradient
  */
-void MainWindow::fillRectWithGradient(QPainter& painter, const QRect& rect, QGradient& gradient)
+void MainWindow::fillRectWithGradient(QPainter& painter, QRect rect, QGradient& gradient)
```

Comment:
I feel like this better stay `const QRect &` since [`QPainter::fillRect`](https://doc.qt.io/qt-5/qpainter.html#fillRect-13) also takes a const reference (and I think Qt API does this in general). But I don't really care so if it's easier to pass by value it's fine with me.

Reply:
I'm fine either way too, it's not a crucial thing.